packages/jsonc/package.json

```
@@ -0,0 +1,30 @@
+{
+  "version": "1.0.0",
+  "name": "@nodejs-loaders/jsonc",
+  "description": "A ESM loader for JSONC files",
+  "type": "module",
+  "author": "Augustin Mauroy",
+  "maintainers": ["Jacob Smith"],
+  "license": "ISC",
+  "main": "./jsonc.mjs",
+  "types": "./jsonc.d.mts",
+  "keywords": [
+    "customisation hook",
+    "loader",
+    "node.js",
+    "json with comments",
+    "jsonc"
+  ],
+  "engines": {
+    "node": ">=20 || >=22 || >=23"
```

Is this equivalent to just

```suggestion
    "node": ">=20"
```

---

Hum why not just `>=20` because we didn't test for 20. So it's kind of lied